view/template/page/forklist.php

```diff
@@ -0,0 +1,25 @@
+<?php Response::setMetaDescription('Subscribe to be notified of fork-related updates') ?>
+<?php Response::setMetaTitle('Blockchain Fork Updates') ?>
+<?php echo View::render('nav/_header') ?>
+<main>
+  <div class="cover cover-dark cover-center cover--dark-overlay" style="background-color:rgb(74, 175, 152)">
```

Comment:
This doesn't have to be a full cover page. I would recommend doing something closer to https://lbry.io/list/subscribe without the sidebar.

Reply:
I copied lbry.io/newslist. Should I also change that one?

Reply:
Possibly, but I don't own that page and I'm not sure how it's being used.

Reply:
I'm gonna leave this as is. Someone will redesign these pages eventually anyway.